chrome/mouse_track.js

```diff
@@ -0,0 +1,63 @@
+/*
```

Comment:
* Most of the functionality seems to be generic, is it really Chrome-specific?
* The file needs a license, have you been able to learn more than:

sure, Formlock is open source - https://github.com/ostarov/Formlock>

Please cite our work properly of course 

* We need to distinguish between yours code and the original